src/management/endpoint.c

```diff
@@ -85,6 +85,10 @@ static int endpoint_create_process_data(struct controller_api *endpoint_create)
 	struct json_object *json_obj = NULL;
 	struct json_object *json_fs_name_obj = NULL;
 	struct json_object *json_endpoint_options_obj = NULL;
+	struct json_object *json_resp_obj = NULL;
+        uint16_t fs_id = 0;
+        struct cfs_fs *fs = NULL;
+        char str[256];
```

Comment:
Please use NAME_MAX macro instead of magic number 256 here. It's defined in `usr/include/linux/limits.h`